### PR TITLE
[next beta]: Bump UI for 1.10

### PR DESCRIPTION
**Pull request by MatApple**
## High Level Description

Update the DC/OS UI package to include the latest fixes and improvements.

Release https://github.com/dcos/dcos-ui/releases/tag/v1.10.0-rc.5

## Related Issues

- [DCOS-17939](https://jira.mesosphere.com/browse/DCOS-17939): Blacklist historySummary. #close
- [DCOS-17527](https://jira.mesosphere.com/browse/DCOS-17527): Update Certified and Community catagory headings. #close
- [DCOS-16599](https://jira.mesosphere.com/browse/DCOS-16599): Introduce hooks to lint commit message. #close
- [DCOS_OSS-1372](https://jira.mesosphere.com/browse/DCOS_OSS-1372): Marathon features list delimiter. #close
- [DCOS-17296](https://jira.mesosphere.com/browse/DCOS-17296): Delete group dialog incorrectly asks you to type a service name instead of a group name. #close
- [DCOS-16744](https://jira.mesosphere.com/browse/DCOS-16744): Add sort for Jobs. #close
- [DCOS-17714](https://jira.mesosphere.com/browse/DCOS-17714): Tests for catalog redirect. #close
- [DCOS-16280](https://jira.mesosphere.com/browse/DCOS-16280): Include forcePull parameter in image object for multi-pod containers. #close
- [DCOS-17361](https://jira.mesosphere.com/browse/DCOS-17361): round cpu allocation used and total value. #close
## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [diff](https://github.com/dcos/dcos-ui/compare/v1.10.0-rc.4...v1.10.0-rc.5)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by MatApple** [comment]
@mesosphere-mergebot bump-ee

**Comment by MatApple** [comment]
@mesosphere-mergebot label [Ready For Review]

File: packages/dcos-ui/buildinfo.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "69815aeeca29f5da76c04e6144705a7d8ce5b2da",
-    "ref_origin": "v1.10.0-rc.4"
+    "ref": "9b25a1c4253f9e48ee58a4341d44ef2167bb484d",
```
Comment (weblancaster) [inline packages/dcos-ui/buildinfo.json]: Looks good https://github.com/dcos/dcos-ui/releases/tag/v1.10.0-rc.5

**Comment by MatApple** [comment]
@mesosphere-mergebot label [Ship It]

**Comment by jgehrcke** [comment]
@orlandohohmeier @daniel0m0baker @MatApple can you please close the corresponding JIRA tickets now that this is merged?

**Pull request merged**